src/Console/MigrateCommand.php

```diff
@@ -4,30 +4,21 @@
 
 namespace LaravelDoctrine\Migrations\Console;
 
-use Illuminate\Console\Command;
 use Illuminate\Console\ConfirmableTrait;
-use LaravelDoctrine\Migrations\Configuration\Configuration;
-use LaravelDoctrine\Migrations\Configuration\ConfigurationProvider;
-use LaravelDoctrine\Migrations\Exceptions\ExecutedUnavailableMigrationsException;
-use LaravelDoctrine\Migrations\Exceptions\MigrationVersionException;
-use LaravelDoctrine\Migrations\Migration;
-use LaravelDoctrine\Migrations\Migrator;
+use LaravelDoctrine\Migrations\Configuration\DependencyFactoryProvider;
 
-class MigrateCommand extends Command
+class MigrateCommand extends BaseCommand
 {
-    use ConfirmableTrait;
-
     /**
      * The name and signature of the console command.
      * @var string
      */
     protected $signature = 'doctrine:migrations:migrate
     {version=latest : The version number (YYYYMMDDHHMMSS) or alias (first, prev, next, latest) to migrate to.}
     {--connection= : For a specific connection }
-    {--write-sql= : The path to output the migration SQL file instead of executing it. }
+    {--write-sql=/tmp : The path to output the migration SQL file instead of executing it. }
```

Comment:
Did we have to set /tmp here? Doctrine fallbacks to getcwd() when this value is null. Hardcoding paths will break for us using open basedir :)